tools/monorepo-utils/src/md-docs/lib/__tests__/generate-manifest.ts

```diff
@@ -137,6 +139,104 @@ describe( 'generateManifest', () => {
 			'https://github.com/edit/example-docs/get-started/local-development.md'
 		);
 	} );
+
+	it( 'should create a hash for each post in a manifest', async () => {
+		const manifest = await generateManifestFromDirectory(
+			rootDir,
+			dir,
+			'example-docs',
+			'https://example.com',
+			'https://example.com/edit'
+		);
+
+		const topLevelCategories = manifest.categories;
+
+		const posts = [
+			...topLevelCategories[ 0 ].posts,
+			...topLevelCategories[ 0 ].categories[ 0 ].posts,
+			...topLevelCategories[ 0 ].categories[ 1 ].posts,
+			...topLevelCategories[ 1 ].posts,
+		];
+
+		posts.forEach( ( post ) => {
+			expect( post.hash ).not.toBeUndefined();
+		} );
+	} );
+
+	it( 'should update a post hash and manifest hash when content is updated', async () => {
+		const manifest = await generateManifestFromDirectory(
+			rootDir,
+			dir,
+			'example-docs',
+			'https://example.com',
+			'https://example.com/edit'
+		);
+		const post = manifest.categories[ 0 ].posts[ 0 ];
+		const originalPostHash = post.hash;
+		const originalManifestHash = manifest.hash;
+
+		// Confirm hashes are not undefined
+		expect( originalPostHash ).not.toBeUndefined();
+		expect( originalManifestHash ).not.toBeUndefined();
+
+		// Update the file content of the corresponding post
+		const filePath = path.join( dir, 'get-started/local-development.md' );
+		const fileContent = fs.readFileSync( filePath, 'utf-8' );
+		const updatedFileContent = fileContent + '\n\n<!-- updated -->';
+		fs.writeFileSync( filePath, updatedFileContent );
+
+		// Generate a new manifest
+		const nextManifest = await generateManifestFromDirectory(
+			rootDir,
+			dir,
+			'example-docs',
+			'https://example.com',
+			'https://example.com/edit'
+		);
+		const nextPost = nextManifest.categories[ 0 ].posts[ 0 ];
+		const NextPostHash = nextPost.hash;
```

Comment:
Unclear why this and the next `const` would start with a capital letter? Is there a reason for this that I'm maybe not aware of?

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/39445/commits/891681e89b5d802769f6770744cbc85cbba328c1